contract/tests/test_contract.py

```diff
@@ -127,41 +134,45 @@ def test_contract_weekly(self):
             [('contract_id', '=', self.contract.id)])
         self.assertTrue(invoices_weekly)
         self.assertEqual(
-            self.contract.recurring_next_date, '2016-03-07')
+            self.contract.recurring_next_date, recurring_next_date)
```

Comment:
You can use `fields.Date.to_date('2016-03-07')` for simplicity (the same for all occurrences)

Reply:
change made.